EIPS/eip-7517.md

```diff
@@ -0,0 +1,112 @@
+---
+eip: 7517
+title: Onchain Consent for AI Data Mining
+description: Bridging the gap between AI workflows and digital asset permissions through a unified blockchain consent.
```

Comment:
To ensure the original intent of the content is maintained, we propose to include "miningPreference" in the metadata. This lets creators control usage in data mining or AI/ML and serves as a guideline, facilitating the ethical and efficient use of content while respecting the creator's rights.

Reply:
Update to "Content mining preference for AI/ML training" to respect the [44-char limit of title in EIP-1](https://eips.ethereum.org/EIPS/eip-1#what-belongs-in-a-successful-eip).

Reply:
For the description, here is the suggested updates: This EIP proposes adding "miningPreference" in the metadata to preserve the digital content's original intent and respect creator’s rights.

Reply:
Update to "A proposal adding "miningPreference" in the metadata to preserve the digital content's original intent and respect creator’s rights." to respect the [140-char limit of description in EIP-1](https://eips.ethereum.org/EIPS/eip-1#what-belongs-in-a-successful-eip).